### DISCUSSION TITLE
[API v3] Get Repo Permissions

### DISCUSSION THREAD

**Discussion by sommersoft** [community/community#24571] [Apps, API and Webhooks]
<p>When calling <span>GET /repos/:owner/:repo</span>, I used to get the following permissions data:</p>
<pre><code>"permissions":{"admin":false,"push":false,"pull":true},
</code></pre>
<p>However, this data is no longer supplied. Was it removed altogether, or is there a newer way to get the information?</p>
<p>Note: it is still listed as a part of the response in the documentation. <a href="https://developer.github.com/v3/repos/#get-a-repository" rel="nofollow noopener">https://developer.github.com/v3/repos/#get-a-repository</a></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/sommersoft">@sommersoft</a>,</p>
<p>Thank you for being here! The permission’s data is included <strong>only</strong> if you authenticate with the API, for example</p>
<pre><code>curl -v -H "Authorization: token $TOKEN" https://api.github.com/repos/github/linguist
</code></pre>
<p>Alternatively, you could use the GraphQL API to fetch a repo and your permission:</p>
<pre><code>query {
	repository(owner:"github", name:"linguist") {
    viewerPermission
  }
}
</code></pre>

**Comment by sommersoft** [comment]
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> ,</p>
<p>Thank you very much for your reply. It highlighted a misstep in my troubleshooting!</p>
<p>I can confirm that it is working with an authenticated request, which my program was using but my manual verification was lacking.</p>
<p>Now on to more debugging…</p>
